movie-browser/src/components/FeatureMovie.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const placeholderImg = 'https://picsum.photos/140/100';
+
+const FeatureMovie = props => {
+  const { movie } = props;
+
+  return (
+    <div>
+      {movie.map(featMovie => (
```

Comment:
what if for some reason `movie` is a falsy value? This component would crash. Either set a defaultProp or use an if to tackle the worse case scenario

Reply:
thanks for pointing this out, added default props to display a generic message if props are not passed to component. Also fixed a bug with movies not having a unique key.